packages/gestalt/src/DropdownSection.js

```diff
@@ -5,12 +5,18 @@ import Text from './Text.js';
 import styles from './Dropdown.css';
 
 type Props = {|
+  /**
+   * Any [Dropdown.Items](https://gestalt.pinterest.systems/dropdown#Dropdown.ItemProps) and/or [Dropdown.Links](https://gestalt.pinterest.systems/dropdown#Dropdown.LinkProps) to be rendered
+   */
   children: Node,
```

Comment:
ditto here on displaying the more restrictive type until we officially support more flexibility

Reply:
DONE